apps/editors/tests/test_views.py

```diff
@@ -1245,6 +1245,16 @@ def test_no_reviews(self):
         eq_(doc('.no-results').length, 1)
         eq_(doc('.review-saved button').length, 1)  # Show only one button.
 
+    def test_do_not_show_reviews_for_unlisted_addons(self):
+        Addon.objects.all().update(status=amo.STATUS_NULL)
```

Comment:
Addons with `STATUS_NULL` aren't unlisted, but incomplete.

To have an Addon unlisted, you need to update its `is_listed` field to `False`:
`Addon.objects.all().update(is_listed=False)`

As a side note, once they're unlisted, they won't be visible using the `Addon.objects` manager, you'll have to use either `Addon.with_unlisted` or `Addon.unfiltered` (which also includes deleted addons, which have `STATUS_DELETED`).